packages/server/test/e2e/1_firefox_spec.ts

```diff
@@ -68,4 +68,12 @@ describe('e2e firefox', function () {
       expect(stderr).not.to.include('foxdriver')
     },
   })
+
+  // NOTE: only an issue on windows
+  // https://github.com/cypress-io/cypress/issues/6392
+  e2e.it.skip('can run multiple specs', {
```

Comment:
can't this test be enabled to run on Windows only?

Reply:
not in our existing test setup, we run 0 e2e tests on Windows

Reply:
ughh, it would be nice to run some e2e tests on Windows